transactions/src/test/java/org/jboss/as/txn/subsystem/TransactionSubsystemTestCase.java

```diff
@@ -172,71 +165,93 @@ public void testAsyncIOExpressions() throws Exception {
 
-        checkSubsystemModelTransformation(mainServices, modelVersion, modelNode -> {
-            modelNode.remove("path");
-            modelNode.remove("relative-to");
-            modelNode.get("process-id-uuid").set(false); //only needs to be removed for 6.2
-            return modelNode;
-        });
-
+        ModelFixer modelFixer = null;
+        switch (controllerVersion) {
+            case EAP_6_4_0:
+                modelFixer = new ModelFixer() {
+                    @Override
+                    public ModelNode fixModel(ModelNode modelNode) {
+                        modelNode.remove("path");
+                        modelNode.remove("relative-to");
+                        return modelNode;
+                    }
+                };
+                break;
+            case EAP_7_0_0:
+            case EAP_7_1_0:
+                modelFixer = new ModelFixer() {
+                    @Override
+                    public ModelNode fixModel(ModelNode modelNode) {
+                        if (modelNode.has("log-store", "log-store")) {
+                            ModelNode logStore = modelNode.get("log-store", "log-store");
+                            if (!logStore.get("expose-all-logs").asBoolean()) {
+                                logStore.remove("expose-all-logs");
+                            }
+                        }
+                        return modelNode;
+                    }
+                };
+                break;
+
+        }
+        checkSubsystemModelTransformation(mainServices, modelVersion, modelFixer);
     }
 
     @Test
-    public void testRejectTransformersEAP620() throws Exception {
-        testRejectTransformers(ModelTestControllerVersion.EAP_6_2_0, MODEL_VERSION_EAP62, new FailedOperationTransformationConfig()
-                .addFailedAttribute(PathAddress.pathAddress(
-                        PathElement.pathElement(SUBSYSTEM, TransactionExtension.SUBSYSTEM_NAME)).append(CMResourceResourceDefinition.PATH_CM_RESOURCE),
-                        FailedOperationTransformationConfig.REJECTED_RESOURCE));
-    }
-
-    @Test
-    public void testRejectTransformersEAP630() throws Exception {
-        testRejectTransformers(ModelTestControllerVersion.EAP_6_3_0, MODEL_VERSION_EAP63, new FailedOperationTransformationConfig()
-                .addFailedAttribute(PathAddress.pathAddress(
-                        PathElement.pathElement(SUBSYSTEM, TransactionExtension.SUBSYSTEM_NAME)).append(CMResourceResourceDefinition.PATH_CM_RESOURCE),
-                        FailedOperationTransformationConfig.REJECTED_RESOURCE));
+    public void testRejectTransformersEAP640() throws Exception {
+        testRejectTransformers(ModelTestControllerVersion.EAP_6_4_0, MODEL_VERSION_EAP64, new FailedOperationTransformationConfig().addFailedAttribute(
+                PathAddress.pathAddress(TransactionExtension.SUBSYSTEM_PATH), new FailedOperationTransformationConfig.NewAttributesConfig("maximum-timeout")));
```

Comment:
Although not caused by this PR, we might as well fix the following. 7.0.0+ introduces the following attributes not found in 6.4.0 (it is in our mail thread): [~maximum-timeout,~ use-journal-store, statistics-enabled, journal-store-enable-async-io]. I crossed out maximum-timeout as that is being handled. But the test xml should be updated to

Reply:
Perhaps we could do that in a follow up issue?

Reply:
Yeah, it is more a bug than anything else. File a high-priority Jira and we're good ;)

Reply:
Thanks Kabir - I filed: https://issues.jboss.org/browse/WFLY-10715